examples/thingProperties.h

```diff
@@ -0,0 +1,34 @@
+#include <ArduinoIoTCloud.h>
+#include <WiFiConnectionManager.h>
+
+char ssid[] = SECRET_SSID;    // Network SSID (name)
+char pass[] = SECRET_PASS;    // Network password (use for WPA, or use as key for WEP)
+
+#define THING_ID "PUT_YOUT_THING_ID_HERE"
+
+void onSwitchChange();
+<<<<<<< HEAD
```

Comment:
It seems that there are more merge conflicts in here - please resolve.
```suggestion
```

Reply:
It was an old example that I put here erroneously.